README.md

```diff
@@ -15,6 +15,12 @@ Customizable and skinnable social platform dedicated to (open)data.
 
 The [full documentation][readthedocs-url] is hosted on Read the Docs.
 
+udata is maintained by [Etalab](https://www.etalab.gouv.fr/), the
```

Comment:
"mainly maintained" maybe?

Reply:
I'd stick to "maintain". "Mainly maintains" is begin for questions on what "mainly" means.

"Maintains" clearly says that Etalab is responsible for stating the roadmap. It does not say that Etalab is the sole contributor nor that Etalab builds the roadmap in its corner. I will add a commit in this PR to explicit this, if you're okay.

Reply:
I am!